packages/instrumentation-sequelize/package.json

```diff
@@ -37,33 +37,27 @@
         "url": "https://github.com/aspecto-io/opentelemetry-ext-js/issues"
     },
     "dependencies": {
-        "@opentelemetry/api": "^0.15.0",
         "@opentelemetry/core": "^0.15.0",
```

Comment:
remove this dependency and add back the dependency on `@opentelemetry/api`

Reply:
fixed